graphql_api/actions/components.py

```diff
@@ -23,3 +27,18 @@ def component_measurements(
         before=before,
         branch=branch,
     )
+
+
+def component_measurements_last_uploaded(
```

Comment:
nit: thoughts on adding return types to these fns? would make it a little easier to review for me (as someone not super versed in python syntax lol)